example_client/cpp/Dockerfile

```diff
@@ -100,7 +100,25 @@ RUN bazel build \
 
 COPY src/ /build/src/
 
-RUN bazel build //src:resnet_client
+RUN bazel build //src:resnet_client_sync
+RUN bazel build //src:resnet_client_benchmark
 
-ENV LD_LIBRARY_PATH=/usr/local/lib
-ENTRYPOINT ["/build/bazel-bin/src/resnet_client"]
+FROM ubuntu:20.04 as release
+
+WORKDIR /clients/
+RUN mkdir -p /clients/libs/ && mkdir -p /clients/images
+COPY --from=base_build \
+    /build/bazel-bin/src/resnet_client_sync \
+    /build/bazel-bin/src/resnet_client_benchmark \
+    /clients/
+COPY --from=base_build \
+    /usr/local/lib/libopencv_core.so* \
+    /usr/local/lib/libopencv_imgcodecs.so* \
+    /usr/local/lib/libopencv_imgproc.so* \
+    /usr/lib/x86_64-linux-gnu/libpng16.so* \
+    /clients/libs/
+
+COPY images/*.jpeg /clients/images/
+COPY input_images.txt /clients/
```

Comment:
images are in different folder so this step fails unless it is copied in advance. It would be easier to download the images via wget https://github.com/openvinotoolkit/model_server/raw/main/example_client/images/airliner.jpeg

Reply:
We copy images dir and input_images.txt in Makefile

Reply:
Done